src/components/Navigation.tsx

```diff
@@ -110,8 +110,9 @@ const Navigation = (props: Props) => {
         <Navbar height='100vh' width={{ sm: 300 }} p="md" fixed>
           <Navbar.Section grow>
             <Group className={classes.header} position="apart">
-              <Text>Ad4min</Text>
-              <Code>v0.0.3</Code>
+              <Image src="Logo44.png"></Image>
+              <Text>AD4Min</Text>
```

Comment:
Is this an expected change? 😄

Reply:
It was on purpose :) But up for discussion.. what do you think?

Reply:
`AD4Min` sound like `AD4M in` (this actually makes sense) or `ad for min` (this is not), a few candidate names for the name changing for discussion,
- **AD4M**
  - this is simple and clean, we can just say to users `download AD4M before using any AD4M apps`, 
  -  the cons is it may be confused with AD4M ontologies itself.
- **AD4M Launcher**
  - this inherits from Holochain Launcher, and makes a lot of sense in holochain ecosystem,
  - but it also may confuse user when the functions of two launchers is different
- **AD4MIN**
  - this is good to reflect our intentions, "when your use AD4MIN, you are the administrator of all AD4M apps".
  - use may think it's a too dev name? 
  - related others
    - `ad4min` is not make `ad4m` uppercase which is inconsistent with `AD4M`
    -  `AD4Min` is a bit misleading when pause words different, 
    - `AD4M IN` looks cool, but I find hard to speak to users, `download AD4M IN before using any AD4M apps`?
-

Reply:
Independent of letters being upper- or lower-case, I always read/say ad4min as "adam-in" since ad4m should be pronounced "adam" (the 4 is mainly for lexical uniqueness).

So I also ready AD4Min as "adamin".

Ok, will go ahead and merge this.